GovUkHtmlHelperExtensions.cs

```diff
@@ -253,15 +253,17 @@ public static class GovUkHtmlHelperExtensions
             this IHtmlHelper<TModel> htmlHelper,
             Expression<Func<TModel, TEnum?>> propertyLambdaExpression,
             FieldsetViewModel fieldsetOptions = null,
-            HintViewModel hintOptions = null)
+            HintViewModel hintOptions = null,
+            string classOptions = null)

```

Comment:
The standard name for this parameter seems to be "classes", please change for consistency

Reply:
Changed